src/lib.rs

```diff
@@ -172,6 +170,12 @@ impl rand::distributions::Sample<usize> for ZipfDistribution {
     }
 }
 
+impl rand::distributions::Distribution<usize> for ZipfDistribution {
```

Comment:
I wonder if we should also implement `IndependentSample` for completeness (even though it is deprecated).